### PR TITLE
Adding validation test for ruleset change PRs & fixing validations in…

### PR DESCRIPTION
**Pull request by Hainish**
… rulesets

### CODE REVIEW

**Comment by Hainish** [comment]
Ready for review - @J0WI @fuglede @jeremyn @cooperq

File: src/chrome/content/rules/opencaching.pl.xml [hunk 1]
```diff
@@ -14,8 +14,5 @@
 	<!-- Forum -->
 	<target host="forum.opencaching.pl" />
 	
-	<!-- Wiki -->
-	<target host="forum.opencaching.pl" />
```
Comment (J0WI) [inline src/chrome/content/rules/opencaching.pl.xml]: should be wiki.opencaching.pl

Comment (Hainish) [inline src/chrome/content/rules/opencaching.pl.xml]: yup I changed and rebased

**Comment by J0WI** [comment]
Just a general question: why are some of the python/shell scripts executable and others just have rw permissions set?

**Comment by Hainish** [comment]
@J0WI I think it's to ensure that the script is called using the `bash` command directly, which suppresses some more verbose output.  Compare `./test/firefox.sh` with `bash test/firefox.sh` for instance.

I thought it was best to preserve permissions for `validate.sh`

**Comment by Hainish** [comment]
Looking into `Travis` failure now.

**Comment by Hainish** [comment]
@J0WI can you merge this in if it looks good?

**Comment by J0WI** [comment]
It lgtm, but I'm not familiar at all with our testing infrastructure.

**Comment by Hainish** [comment]
It's changed and I don't know if anyone who is available is fully familiar with it.  Merging due to the fact that this is high-priority.

**Comment by cooperq** [comment]
lgtm

**Pull request merged**